velox/exec/GroupingSet.cpp

```diff
@@ -356,11 +379,13 @@ const SelectivityVector& GroupingSet::getSelectivityVector(
 
 bool GroupingSet::getOutput(
     int32_t batchSize,
-    bool isPartial,
-    RowContainerIterator* iterator,
+    RowContainerIterator* FOLLY_NONNULL iterator,
```

Comment:
getGlobalAggregationOutput was modified to take iterator by reference; let's make the same change here for consistency